preprocessing/lib/load-data.js

```diff
@@ -1,3 +1,8 @@
+/*
+ * Loads regions from CSV files, corrects their names, and returns them
+ * as JSON.
```

Comment:
JSON is a file format, not a data format, instead of saying "as JSON" describe the hierarchy this script returns (or move an existing definition of this hierarchy in here)

Reply:
Resolved in 45d3f2619f2e382c7871963f28f7788f3321ed40